Automattic/jetpack
# Subscription Site: Update Subscribe block after the post nudge

Part of https://github.com/Automattic/jetpack/issues/36021

## Proposed changes:

- updates the nudge for the Subscribe block
- adds support for inherit anchor bock layout

### Block theme (Twenty Twenty-Three)

<img width="678" alt="Screenshot 2024-03-06 at 13 15 19" src="https://github.com/Automattic/jetpack/assets/4068554/771d536b-e9af-41a3-a228-f5aa089a9d1c">

### Classic theme (Twenty Ten)

<img width="997" alt="Screenshot 2024-03-06 at 13 14 14" src="https://github.com/Automattic/jetpack/assets/4068554/0c4f7851-12c7-4427-8488-283898f60d47">

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






### Prerequisites

- enable the Subscription Site feature by adding the following line to your `0-sandbox.php` file:
```php
add_filter( 'jetpack_subscription_site_enabled', '__return_true' );
```
- enable the Jetpack Newsletter Module
- go to Jetpack Newsletter Settings and enable automatic insertion of the Subscribe block at the end of each post

### Testing scenarios

- make sure the Subscribe block after the post looks good for block themes
- make sure the Subscribe block after the post looks good for classic themes
